ttnn/ttnn/operations/pool.py

```diff
@@ -137,7 +137,13 @@ def _golden_function(input_tensor: ttnn.Tensor):
     ttnn._ttnn.operations.pool.global_avg_pool2d
 )
 
-max_pool2d = ttnn._ttnn.operations.pool.max_pool2d
-average_pool_2d = ttnn._ttnn.operations.pool.average_pool_2d
+max_pool2d = ttnn.register_operation(
+    name="ttnn.max_pool2d", is_method=True, validate_input_tensors=lambda *args, **kwargs: None
```

Comment:
do you actually have to specify is_method and validate_input_tensors? I think those can go